### DISCUSSION TITLE
postgres_up not available

### DISCUSSION THREAD

**Discussion by Alexdrr** [weaponry/pgscv#20] [General]
Hi, I have configured the documentation exporter, work great but when i do killall -9 postgres, I get a message in the logs  `(dial tcp 127.0.0.1:5432: connect: connection refused), skip collect"` , how can i get the metric  `postgres_up == 0` in this case? 
I check [postgres_exporter](https://github.com/prometheus-community/postgres_exporter) he doesn't have such a problem

**Comment by lesovsky** [comment]
Hi, currently it is no implemented. You could make an issue and i will add this.
